src/wms/src/main/java/org/geoserver/wms/capabilities/GetCapabilitiesTransformer.java

```diff
@@ -1033,6 +1034,14 @@ protected void handleLayer(final LayerInfo layer, boolean isRoot) throws IOExcep
                 handleRootCrsList(srsList);
             } else {
                 element("SRS", srs);
+                // overridingServiceSRS
```

Comment:
What's going on here?

Reply:
advertising additionally supported SRS in WMS capability doc, as per our dicussion

Reply:
No idea what you are talking about, This PR is about formats and styles.

Reply:
ok this will be reverted

Reply:
ok will remove it